### PR TITLE
feat: add meta data to cli output

### PR DESCRIPTION
**Pull request by jkodu**
Log metadata of test engine, environment and runner to cli

image

**Note: to add tests, hence still a draft**

Closes issue:
- https://github.com/dequelabs/attest-issues/issues/6
- https://github.com/dequelabs/axe-cli/issues/93 (duplicate)

## Reviewer checks

**Required fields, to be filled out by PR reviewer(s)**
- [ ] Follows the commit message policy, appropriate for next version
- [ ] Has documentation updated, a DU ticket, or requires no documentation change
- [ ] Includes new tests, or was unnecessary
- [ ] Code is reviewed for security by: << Name here >>

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,6 +1,9 @@
 'use strict';
 
 const program = require('commander');
+
+const prettyjson = require('prettyjson');
```
Comment (stephenmathieson) [inline index.js]: Please remove these newlines.

Also, not sure I think this is "pretty json" 🤔. I think the left-hand side should be displayed differently than the right. It'll make the relevant stuff "stick out" more.

**Comment by stephenmathieson** [comment]
We should also make sure the metadata is outputted in the JSON mode.

**Comment by jkodu** [comment]
Ensured that this metadata is outputted in JSON mode as well by running `axe https://deque.com --stdout` & got the below.

image

**Comment by dylanb** [comment]
reviewed for security retroactively

**Pull request merged**